### PR TITLE
Iosxe/show wireless stats ap join summary

### PR DESCRIPTION
**Pull request by robschn**

### CODE REVIEW

**Review by badalsoyantar — APPROVED** [review]

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py
Comment (robschn) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py:294]: I'll look into why the parser is adding `" "`

Comment (robschn) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py]: I was using `pprint()` during testing and it added the extra `" "`. Fixed now! Waiting on review from @tahigash

File: src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessStatsApJoinSummary/cli/equal/golden_output_expected.py:1]: schema looks good

**Review by tahigash — APPROVED** [review]

**Comment by tahigash** [comment]
thank you for your contribution!

**Pull request merged**
